### ISSUE TITLE
Capella Simplified Metamodel - Appearance & Style properties

### ISSUE THREAD

**Issue by mvccogo** [labs4capella/python4capella#148]
Greetings! According to the Simplified Metamodel, "...the content of diagram (graphical elements) is not yet defined". Does this mean that Style & Appearance properties cannot be exported/imported? If so, how could I add support for real-time editing of those elements? Below, the properties that I need to edit.

![image](https://user-images.githubusercontent.com/20509814/189486341-e9a93d8d-357d-4aaf-90b8-9abdadcfd521.png)

**Comment by mvccogo** [comment]
This seems related to #51, I'm new to Eclipse/Java development but willing to add this feature if needed.

**Comment by ylussaud** [comment]
No, for the moment we don't provide an API to get/set colors and styles. And yes it's related to #51. There is a [discussion](https://forum.mbse-capella.org/t/editing-block-format-with-python4capella/5661) on on how to do this on the Capella forum. And also [one](https://forum.mbse-capella.org/t/graphical-coordinates-of-a-capella-element-inside-a-diagram-node-edge/5646) on getting GMF Notation Nodes to get the coordinates of an element on a diagram that might be useful as well for diagram manipulation.

**Comment by mvccogo** [comment]
@ylussaud I see... I will give Stephane Lacrampe's solution a try, maybe that approach will be enough for what I need.

Having said that, if someone were to develop the necessary classes to directly deal with GMF definitions (in the Sirius module along with additions to the simplified Python API as you suggested), would that be implemented in P4C? Or is that considered too specific/out of scope of P4C?

**Comment by ylussaud** [comment]
It should be in the scope of #51, but I don't know if it will expose the GMF Notation concepts or provide shortcuts to deal with some of its properties...
